src/CameraComponent.h

```diff
@@ -68,6 +70,10 @@ class CameraComponent {
                          size_t value_size, int param_type);
     virtual int setCameraMode(uint32_t mode);
     virtual int getCameraMode();
+    virtual int startImageCapture(int interval, int count,
+                                  std::function<void(int result, int seq_num)> cb);
```

Comment:
You can define callback type like this, 
`typedef std::function<void(int result, int seq_num)> image_capture_callback_t;`

Reply:
Fixed